.pipelines/azure-pipelines/azure-pipelines-REFramework.yml

```diff
@@ -0,0 +1,36 @@
+trigger:
+  batch: true
+  branches:
+    include:
+    - develop
+    - master/REFramework/*
+  paths:
+    include:
+    - REFramework/*
+    - .pipelines/*
+
+
+pr: none
+  # branches:
+  #   include:
+  #   - develop
+  #   - master/REFramework/*
+  # paths:
+  #   include:
+  #   - REFramework/*
+  #   - .pipelines/*
+
+
+variables:
+  - template: .pipelines/variables/common.yml
```

Comment:
nok, location is relative to your current location
../variables/common.yml

Reply:
fixed